sample.py

```diff
@@ -7,12 +8,18 @@
 
 
 def sample(model_path: str, num_samples: int = 1000):
+    kwargs = DistributedDataParallelKwargs(find_unused_parameters=True)
+    accelerator = Accelerator(
+        kwargs_handlers=[kwargs],
+        split_batches=True,
+        log_with=["wandb"],
+    )
     # Instantiating data and model
 
     print("Loading data")
-    encode_data, _ = load_data(
-        data_path="dnadiffusion/data/K562_hESCT0_HepG2_GM12878_12k_sequences_per_group.txt",
-        saved_data_path="dnadiffusion/data/encode_data.pkl",
+    encode_data = load_data(
+        data_path="src/dnadiffusion/data/K562_hESCT0_HepG2_GM12878_12k_sequences_per_group.txt",
```

Comment:
This should be a separate issue, but we should just make a top-level `data/` folder or specify an external source rather than keeping these in a `src/` subfolder.